lib/llm/src/kv_router.rs

```diff
@@ -42,11 +43,16 @@ use crate::{
         scheduler::{KvScheduler, KvSchedulerError, SchedulingRequest},
         scoring::ProcessedEndpoints,
     },
+    preprocessor::BackendInput,
+    protocols::common::llm_backend::LLMEngineOutput,
     tokens::TokenBlockSequence,
 };
 
 use dynamo_runtime::traits::events::EventSubscriber;
 
+// TODO: Allow user to change
+pub const DEFAULT_KV_BLOCK_SIZE: usize = 16;
```

Comment:
Not at my computer to search right now, but I also thought this was already defined somewhere on rust side too.

Also, @GuanLuo @alec-flowers did we have some way to make sure rust side router/scheduler and publisher side were using the same block size on both sides? IIRC there was some silent issues when they mismatched.